src/vs/workbench/contrib/terminalContrib/suggest/browser/terminal.suggest.contribution.ts

```diff
@@ -190,7 +190,8 @@ class TerminalSuggestContribution extends DisposableStore implements ITerminalCo
 		this.add(textVirtualModel);
 
 		const virtualProviders = this._languageFeaturesService.completionProvider.all(textVirtualModel.object.textEditorModel);
-		const provider = virtualProviders.find(p => p._debugDisplayName === PYLANCE_DEBUG_DISPLAY_NAME);
+		// TODO: Remove hard-coded filter for Python REPL.
```

Comment:
may want to make an issue for this?

Reply:
Sounds good! Tracking them here: https://github.com/microsoft/vscode/issues/257343